Gemfile

```diff
@@ -159,6 +167,11 @@ group :development, :test do
   gem 'pry-rails', '~> 0.3.6'
   gem 'awesome_print'
   gem 'puma', '~> 5.3'
+  # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+  # We may eventually remove the pry-* gems in favor of 'debug'.
+  # Installation results in an intermittent spec failure, so commenting this out for now:
+  # https://cm-jira.usa.gov/browse/SRCH-3344
+  # gem 'debug', platforms: %i[mri mingw x64_mingw]
```

Comment:
About to merge `main`, which will install `debug` now that SRCH-3344 is resolved.